### PR TITLE
[ADD] Add account_statement_reopen module

### PR DESCRIPTION
**Pull request by adrienpeiffer**
This module allows to reopen a bank statement without canceling.

### CODE REVIEW

File: account_statement_reopen/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': "Account statement reopen",
+
```
Comment (thomaspaulb) [inline account_statement_reopen/__openerp__.py:6]: Why a space here?

File: account_statement_reopen/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    'summary': """
```
Comment (thomaspaulb) [inline account_statement_reopen/__openerp__.py:7]: And why a return here? :-)

File: setup/account_statement_reopen/odoo_addons/__init__.py [hunk 3]
```diff
@@ -0,0 +1 @@
+__import__('pkg_resources').declare_namespace(__name__)
```
Comment (thomaspaulb) [inline setup/account_statement_reopen/odoo_addons/__init__.py]: Just out of curiosity, what do these `setup/` files exactly do? I haven't seen them in any Odoo module so far, so do we really need them?

**Review by thomaspaulb — APPROVED** [review]
I tested the module and it works well, it saves us a lot of work so I would really love to see this in the main OCA 8.0 branch. I only have some minor style comments, but nothing serious, so I approve this module.

**Comment by pedrobaeza** [comment]
@sbidoul should we close this?

**Pull request closed without merging**